controllers/rcj_soccer_referee_supervisor/referee/progress_checker.py

```diff
@@ -43,15 +38,14 @@ def track(self, position):
 
         self.prev_position = position
 
-    def is_progress(self, robot=None):
+    def is_progress(self, robot=None) -> bool:
```

Comment:
@Adman I am wondering whether this ought to actually be a property rather than a function.

What do you think?

In any case, the docstring is missing a `Returns` section, that's something I should probably add on the next pass...

Reply:
It cannot be a property since it takes `robot` argument.

Reply:
@Adman sorry, you are very much right! What (I guess) I was alluding to was a complete overhaul of the architecture, which would allow us to do it the other way around and have things like `robot.is_progressing` (or `self.is_progressing`) and so on.

That will have to wait for the architecture revamp though.